.github/workflows/ci-code-review-rust.yml

```diff
@@ -52,17 +47,17 @@ jobs:
         with:
           toolchain: stable
           profile: minimal
-          components: rustfmt
-          override: true
+          components: rustc
+
+      - name: Cache dependencies
+        uses: Swatinem/rust-cache@v2
 
       - name: Install Solana
-        run: |
-          sh -c "$(curl -sSfL https://release.solana.com/v${{ env.SOLANA_VERSION }}/install)"
-          echo "$HOME/.local/share/solana/install/active_release/bin" >> $GITHUB_PATH
-          export PATH="/home/runner/.local/share/solana/install/active_release/bin:$PATH"
-          solana --version
-          echo "Generating keypair..."
-          solana-keygen new -o "$HOME/.config/solana/id.json" --no-passphrase --silent
+        uses: metadaoproject/setup-solana@v1.0
```

Comment:
nice addition. Should we specified the versions needed for anchor and solana as documented here https://github.com/metaDAOproject/anchor-test#configuring-versions ?

Reply:
This is a good point, sir. It is now corrected